src/lib/button/button.ts

```diff
@@ -157,6 +157,14 @@ export class MdButton implements OnDestroy {
   _isRippleDisabled() {
     return this.disableRipple || this.disabled;
   }
+
+  _checkElementAttribues(list: Array<string>) {
```

Comment:
I spent almost 5 minutes figuring out what i want to call it, eventually i gave up and assumed you'd have a better name :) thanks